spec/travis_migrations_spec.rb

```diff
@@ -1,33 +1,39 @@
 require 'spec_helper'
 require 'yaml'
 
-
 describe 'Rake tasks' do
   let(:config) { YAML.load(ERB.new(File.read('config/database.yml')).result) }
   let(:conn)   { ActiveRecord::Base.connection }
   let(:tables) { conn.select_values("SELECT table_name FROM information_schema.tables WHERE table_schema = 'public'") }
   before       { ActiveRecord::Base.establish_connection(config['test']) }
 
   def run(cmd)
-    system "RAILS_ENV=test bundle exec #{cmd}"
+    system "RAILS_ENV=test LOGS_DATABASE=#{ENV['LOGS_DATABASE'].to_i} bundle exec #{cmd}"
```

Comment:
It inherits the `LOGS_DATABASE` env var from `.travis.yml` and passes it through to the sub-process -- both are covered.